apps/frontend/assets/src/omg-page-user-setting/SecuritySection.js

```diff
@@ -0,0 +1,53 @@
+import React, { useCallback } from 'react'
+import styled from 'styled-components'
+import { useDispatch, useSelector } from 'react-redux'
+import { selectCurrentUser } from '../omg-user-current/selector'
+import { Button } from '../omg-uikit'
+import { openModal } from '../omg-modal/action'
+
+const SecurityContainer = styled.div`
+  h3 {
+    margin: 20px 0;
+  }
+  .enable-two-fa {
+    margin-left: 20px;
+  }
+`
+function SecuritySeciton () {
+  const currentUser = useSelector(selectCurrentUser)
+  const dispatch = useDispatch()
+  const onClickEnable2Fa = useCallback(
```

Comment:
why do you need useCallback here?

Reply:
not really necessary, just some perf for the page.